Makefile

```diff
@@ -663,3 +663,12 @@ build-matrix.html: $(foreach PKG,$(PKGS), $(TOP_DIR)/src/$(PKG).mk)
 	@echo '</table>'                        >> $@
 	@echo '</body>'                         >> $@
 	@echo '</html>'                         >> $@
+
+
+versions.json: $(foreach PKG,$(PKGS), $(TOP_DIR)/src/$(PKG).mk)
+	@echo '{'                         > $@
+	@{$(foreach PKG,$(PKGS),          \
+	    echo -e '\t "$(PKG)":         \
+	        "$($(PKG)_VERSION)",';)} >> $@
+	@echo -e '\t "": null'           >> $@
```

Comment:
It is. This is just an ugly workaround because the last comma would produce a syntax error instead... If you have a better idea...